feat: style improvements

fairlighteth opened the pull request:
# Summary

- Adds a pending message in the signing state:


https://github.com/cowprotocol/cowswap/assets/31534717/4ba42b49-6854-46a6-b2a4-0288620dadd2

shoom3301 commented on apps/cowswap-frontend/src/common/pure/TradeLoadingButton/index.tsx:
```diff
@@ -3,31 +3,10 @@ import { useEffect, useState } from 'react'
-
-  ${fadeIn}
-`
+import { CenteredDots, LongLoadText } from 'modules/trade/pure/TradeConfirmation/styled'
```
Common things should not depend on modules.
Would be event better to move the component into `@cowprotocol/ui`

fairlighteth replied:
@shoom3301 addressed this now

elena-zh commented:
Hey @fairlighteth, great job!
Question: there are loaders instead of the message in the Classic eth-flow modal. Is there possibility to fix this in the current PR or it is better to create a separate issue for this? 
![approving](https://github.com/cowprotocol/cowswap/assets/70885163/16978dee-2bbe-43d1-80fd-0ffe8dcc90a1)
![classic eth](https://github.com/cowprotocol/cowswap/assets/70885163/6615c47d-0472-4c6b-9bb4-8d997fbc2e56)
![Screenshot_1](https://github.com/cowprotocol/cowswap/assets/70885163/db25bc58-5671-412a-bdbf-92bad9292b8c)

Thanks!

fairlighteth commented:
@elena-zh Addressed all points in this PR:


https://github.com/cowprotocol/cowswap/assets/31534717/6244c8e7-a8c5-4889-8d45-588657fb8484


https://github.com/cowprotocol/cowswap/assets/31534717/2843b536-fdda-4094-87ad-b41c60abf0a6

elena-zh commented:
Hey @fairlighteth , great, thank you!
One proposal from my side is to change 'Sing' to 'Confirm' word for the Classic eth-flow modals (as these are onchain TXs). WDYT? :)
![confirm](https://github.com/cowprotocol/cowswap/assets/70885163/d339a408-8601-4108-a2c7-ca71abcd28b0)

elena-zh reviewed (approved):
Great, thank you!
